src/themes/mainTheme.ts

```diff
@@ -37,18 +35,19 @@ export const mainTheme = createTheme(
         contrastText: Color.White,
       },
       secondary: {
-        main: Color.SecondaryMain,
+        light: '#FEFBF3',
+        main: '#F9F4E6',
+        dark: '#EDE2C7',
       },
       error: {
         main: Color.ErrorMain,
-        light: Color.ErrorLight,
       },
       success: {
         main: Color.SuccessMain,
       },
       background: {
         default: Color.White,
-        paper: Color.Paper,
+        paper: '#FEFBF3',
```

Comment:
Kanskje vi bare skulle flytta alle hex-verdiene til Color igjen likevel. Da blir det mye enklere å se at paper faktisk er det samme som secondary.light

Reply:
Ah, ser nå at du faktisk har gjort det :D bare bruk de i stedet for hex-verdiene igjen her og i secondary da